### PR TITLE
Working

### PR DESCRIPTION
**Pull request by BSGreaves**
@drdreed
This one's ready too.
I just noticed I left a console.log on line 2... oops. Will remove it when I get your other feedback on it.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var alphabet = "abcdefghijklmnopqrstuvwxyz".split("");
+console.log(alphabet);
```
Comment (DevDreed) [inline main.js]: remove this console.log

File: main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+var output = "";
+
+function stackLetters (array) {
+     for (var i = 0; i < array.length; i++) {
+     	output += array[i];
+     	console.log(output);
+	}
```
Comment (DevDreed) [inline main.js]: Make sure you format your document before making a pull request.  Use a plugin to beautify your code.

**Comment by BSGreaves** [comment]
@drdreed Code properly indented and removed the extra console.log. Thx.

**Review by DevDreed — APPROVED** [review]
👍

**Pull request merged**